interactions/__init__.py

```diff
@@ -110,10 +111,29 @@ def __init__(self, name=None,
             if item.antiparticle:
                 continue
 
+            if item.specie.name == 'Electron':
```

Comment:
Random constant inside the code, I'm strongly against it :) 

First, what's the problem? Do you want to output integrals while debugging and there are too many of them? Then make a function in `utils.py` that will output what you want. Don't add random exclusions all around the code. You made a non-local change (in multiple places) to fix a tiny convenience problem

Reply:
PS; I removed collision integrals for electron, since they became distracting.

Will take care of this I guess.

Reply:
I'm also against that — see other comment :)